packages/tenant-process/test/db.test.ts

```diff
@@ -78,6 +103,784 @@ describe("database test", () => {
     });
   });
   describe("readModelService", () => {
+    describe("getConsumers", () => {
+      const organizationProducerId = uuidv4();
+      const consumerId1 = uuidv4();
+      const consumerId2 = uuidv4();
+      const consumerId3 = uuidv4();
+      const mockTenant = getMockTenant();
+
+      const tenant1: Tenant = {
+        ...mockTenant,
+        id: consumerId1,
+        name: "A tenant1",
+      };
+      const tenant2: Tenant = {
+        ...mockTenant,
+        id: consumerId2,
+        name: "A tenant2",
+      };
+      const tenant3: Tenant = {
+        ...mockTenant,
+        id: consumerId3,
+        name: "A tenant3",
+      };
+
+      it("should get the tenants consuming any of the eservices of a specific producerId", async () => {
+        await addOneTenant(tenant1, postgresDB, tenants);
+
+        const descriptor1: Descriptor = {
+          ...mockDescriptor,
+          state: descriptorState.published,
+        };
+
+        const eService1: EService = {
+          ...mockEService,
+          id: uuidv4(),
+          name: "A",
+          descriptors: [descriptor1],
+          producerId: organizationProducerId,
+        };
+        await addOneEService(eService1, eservices);
+
+        const agreementEservice1 = getMockAgreement({
+          eServiceId: eService1.id,
+          descriptorId: descriptor1.id,
+          producerId: organizationProducerId,
+          consumerId: consumerId1,
```

Comment:
```suggestion
          producerId: eService1.producerId,
```
There's no need for setting the consumerId, it always generetes a new one with the change in `getMockAgreement`